rust/agama-network/src/nm/dbus.rs

```diff
@@ -71,7 +71,8 @@ pub fn merge_dbus_connections<'a>(
 ) -> NestedHash<'a> {
     let mut merged = HashMap::new();
```

Comment:
also here it can use capacity
```suggestion
    let mut merged = HashMap::with_capacity(original.len());
```